polaris-react/src/components/AlphaFilters/AlphaFilters.tsx

```diff
@@ -73,6 +77,8 @@ export interface AlphaFiltersProps {
   focused?: boolean;
   /** Available filters added to the filter bar. Shortcut filters are pinned to the front of the bar. */
   filters: FilterInterface[];
+  /** Additional filters organized by sections. */
+  additionalFilterSections?: FilterSectionInterface[];
```

Comment:
This "additional" prefix sounds like they're supplemental to those provided to the `filters` prop instead of being either/or. What do you think about naming it just `sectionedFilters` or `filterSections`? The filters prop will also need to be made optional support/clarify this like in ActionList.

```suggestion
  /** Filters organized by section. */
  filterSections?: FilterSectionInterface[];
```